bookkeeper-server/src/main/java/org/apache/bookkeeper/server/http/service/TriggerGCService.java

```diff
@@ -61,40 +62,46 @@ public TriggerGCService(ServerConfiguration conf, BookieServer bookieServer) {
     @Override
     public HttpServiceResponse handle(HttpServiceRequest request) throws Exception {
         HttpServiceResponse response = new HttpServiceResponse();
+        try {
+            if (HttpServer.Method.PUT == request.getMethod()) {
+                String requestBody = request.getBody();
+                if (StringUtils.isBlank(requestBody)) {
+                    bookieServer.getBookie().getLedgerStorage().forceGC();
+                } else {
+                    @SuppressWarnings("unchecked")
+                    Map<String, Object> configMap = JsonUtil.fromJson(requestBody, HashMap.class);
+                    Boolean forceMajor = (Boolean) configMap.getOrDefault("forceMajor", null);
```

Comment:
Shouldn't the default value be a boolean? If it is null, it would be easy to cause NPE in the inner code. We can use Boolean.parseBoolean() to make sure it is a boolean and avoid a cast error happening.

Reply:
Done.